article/app/model/dotcomponents/DotcomponentsCommercialHelper.scala

```diff
@@ -0,0 +1,34 @@
+package model.dotcomponents
+
+import common.Edition
+import model.{ApplicationContext, PageWithStoryPackage}
+import play.api.libs.json.{JsBoolean, Json}
+import play.api.mvc.RequestHeader
+import views.support.JavaScriptPage.getMap
+
+case class CommercialConfiguration(
```

Comment:
Should we rename the file to `CommercialConfiguration.scala` now that it's just the case class and companion object?

Reply:
Done. Thanks!